src/transformers/trainer_utils.py

```diff
@@ -250,6 +250,9 @@ def speed_metrics(split, start_time, num_samples=None):
     if num_samples is not None:
         samples_per_second = 1 / (runtime / num_samples)
         result[f"{split}_samples_per_second"] = round(samples_per_second, 3)
+    if num_steps is not None:
+        steps_per_second = 1 / (runtime / num_steps)
```

Comment:
```suggestion
        steps_per_second = num_steps / runtime
```

Won't this be easier to read ;)

same for `num_samples` above?

Reply:
I just copy-pasted without thinking ;-) This is better indeed, will do the other one.